SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -64,15 +64,14 @@ Development files for %{name}
     -DFLB_DEBUG=Off \
     -DFLB_TLS=On \
     -DFLB_JEMALLOC=On \
-    -DFLB_LUAJIT=Off \
```

Comment:
I was thinking you wanted to also -DFLB_PREFER_SYSTEM_LIBS=On and then BuildRequires Luajit-devel as was done here https://github.com/microsoft/azurelinux/pull/11009

Was that considered?

Reply:
discussed offline.  Resolving issue.